spec/emittr_spec.rb

```diff
@@ -77,6 +77,29 @@
     end
   end
 
+  describe '#once' do
+    let(:callback) { Proc.new {} }
+
+    before { emitter.once :once_test, &callback }
+
+    it 'calls callback' do
+      expect(callback).to receive(:call)
+      emitter.emit :once_test
+    end
+
+    it 'adds listeners to list when not emitted yet' do
+      listeners = emitter.send(:listeners)
+      expect(listeners[:once_test]).to be_any
```

Comment:
yeah, the `be_any` matcher makes it very "unclear". I'll think in a better expectation for this